docker-compose.yml

```diff
@@ -1,25 +1,26 @@
 version: '3'
 services:
   app:
+    ## Build to use locak Dockerfile
+    # build: .
+    ## Use image to to pull from docker
     image: bpanel/bpanel
     restart: unless-stopped
     ports:
       - "5000:5000"
       - "8000:8000"
     env_file:
-      - ./bcoin.env
       - ./secrets.env
     environment:
-      BCOIN_HOST: bcoin
-    ## Uncomment to build
-    # build: .
+      BPANEL_HOST: bcoin
+      BPANEL_NETWORK: regtest
     volumes:
       ## Mapping to local files- Handy for dev
-      ## Uncomment for server dev
+      - ./webapp:/usr/src/app/webapp
       - ./server:/usr/src/app/server
       # - ./scripts:/usr/src/app/scripts
-      ## Uncomment for webapp dev
-      - ./webapp:/usr/src/app/webapp
+      # Use below to copy local bpanel config files
+      - ~/.bpanel:/root/.bpanel
```

Comment:
I also had reservations about putting it in the home directory, but there are a couple reasons I went in this direction: 
1) We don't want to put configs in `./config` because I'd rather not have to either a) check these into version control or b) avoid checking in changes to version control. 
2) I think it's best to stick with the bcoin conventions for configs, especially since we're using bcfg. Moving things to a relative path will really mess with the way bcfg looks for config files, even with the prefix option set, it's not very straightforward.

I've actually been thinking about moving more out of the local config directory, including the pluginsConfig. Moving them out and probably also into ~/.bpanel avoids some of the frustrations of developing to make sure you don't check in changes to the config when playing with plugins and is incidentally the same convention hyper.is uses. 

As for helping the first time dev experience, the problem with giving config files that are checked in is that everyone will have different nodes that they want to connect to. If someone wants to get an environment working quickly out of the box, `docker-compose up -d` still works and gets you connected to a local regtest node. One way I expect to make this even easier in the future is to have a startup screen for the UI where you can put in configuration information if none is found on startup to point to a node. Basically, this is part of moving in a direction where bPanel is much more decoupled from bcoin which is going to be much closer to real-world conditions. 

I'm not 100% thrilled with this approach, and have some of the same hesitations as you I think, but ultimately I think the trade-offs are worth it.

Reply:
I think bcoin decided to use the home dir for the sake of simplicity, but I don't think JJ would have a problem with adding a "config-prefix" flag.

Reply:
There already is a prefix flag, so if you wanted to add configs somewhere else too, you can do that, it just looks in the home directory first and it's hard coded in ([here](https://github.com/bcoin-org/bcfg/blob/master/lib/config.js#L721) and [here](https://github.com/bcoin-org/bcfg/blob/master/lib/config.js#L32)).

It is possible to load configs from a custom file/directory. So we could have an added option that can be passed at runtime like "custom-config" where, when configs are loaded it looks there. I just think this starts to add a lot of complexity where the prefix, env, and args already give some flexibility. Also, I do think there's something to be said for keeping with the bcoin convention (and frankly the hyper.is one too)

Reply:
oh, well great!  I still think we should not remove the default symnet configs for developers though, so maybe we do the `./config:/root/.bpanel` thing and provide example configs?  Devs still need to be able to spinup bcoin+pbanel nodes with different configs easily, and to do that we need to check in example configs.

Reply:
I added an `/etc` directory with sample configs :)